pages/api/tasks/[taskId].ts

```diff
@@ -0,0 +1,28 @@
+import nextConnect from 'next-connect';
+import { NextApiRequest, NextApiResponse } from 'next';
+import middleware from '../../../middlewares/middleware';
+import { Document } from 'mongoose';
+import Task from '../../../models/task/task.schema';
+import Solution from '../../../models/solution/solutions.schema';
+
+const handler = nextConnect();
+handler.use(middleware);
+
+handler.get(
+  async (req: NextApiRequest, res: NextApiResponse<Document[]>): Promise<void> => {
+    try {
+      const task = await Task.findById(req.query.taskId).populate('_user');
```

Comment:
to wszystko tu ulegnie zmianie jak się pozbedziemy mongoosa i tak

Reply:
Spoko... ;)